### PR TITLE
Fix for exception when serializing element dictionaries with INFINITE…

### PR DESCRIPTION
**Pull request by JonGabilondoAngulo**
… values. Targets dependencies on RoutingHTTPSever.framework fixed to get WDARunner signature correctly.

Found at least a case of elements with INFINITE values in x,y. In Springboard widgets screen, at the bottom there are some "Link" elements with INFINITE x,y. This breaks the serialisation of the elements dictionary.

On the other hand, a more general problem is that RoutingHTTPServer.framework is not code signed because it is embedded into the WebRiverAgentLib framework. This is a documented problem and the solution is to make the Runner know that the framework exists.

### CODE REVIEW

File: WebDriverAgentLib/Categories/XCUIElement+FBWebDriverAttributes.m [hunk 1]
```diff
@@ -161,4 +161,15 @@ - (NSDictionary *)wdRect
   };
 }
 
+- (NSDictionary *)wdRectNoInf
+{
+  CGRect frame = self.wdFrame;
+  return @{
+           @"x": @((frame.origin.x != INFINITY) ?frame.origin.x :0),
```
Comment (mykola-mokhnach) [inline WebDriverAgentLib/Categories/XCUIElement+FBWebDriverAttributes.m]: such values are returned by XCTest and I don't think it makes a lot of sense to build such workarounds over them. I'd rather try to find the reason of why it happens. Perhaps, it is some wrong way of reading element attributes in WDA source or a bug on the Apple side?

Comment (JonGabilondoAngulo) [reply WebDriverAgentLib/Categories/XCUIElement+FBWebDriverAttributes.m]: My debugging says it's an Apple bug. When the Links are not visible, at least the ones at the bottom of the Springboard, they show the INFINITE values. I need the springboard tree, so this patch helps.

Comment (mykola-mokhnach) [reply WebDriverAgentLib/Categories/XCUIElement+FBWebDriverAttributes.m]: why not to use `isinf`?

File: WebDriverAgent.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -2079,10 +2092,12 @@
 			baseConfigurationReference = EEE5CABF1C80361500CBBDD9 /* ProjectSettings.xcconfig */;
 			buildSettings = {
 				CLANG_STATIC_ANALYZER_MODE = deep;
+				CODE_SIGN_IDENTITY = "iPhone Developer";
```
[the remark this answers is not in the document]
Comment (JonGabilondoAngulo) [reply WebDriverAgent.xcodeproj/project.pbxproj]: It shouldn't sorry.

File: WebDriverAgentLib/Categories/XCUIElement+FBWebDriverAttributes.m (hunk 1 above)
Comment (JonGabilondoAngulo) [reply WebDriverAgentLib/Categories/XCUIElement+FBWebDriverAttributes.m]: Nicer with isinf, thanks.

**Pull request closed without merging**